src/components/SpruceForm/FieldTemplates.tsx

```diff
@@ -30,9 +91,12 @@ export const ArrayFieldTemplate: React.FC<ArrayFieldTemplateProps> = ({
 }) => {
   const id = idSchema.$id;
   const description = uiSchema["ui:description"] || schema.description;
-  const buttonText = uiSchema["ui:buttonText"] || "Add";
+  const addButtonSize = uiSchema["ui:addButtonSize"] || "small";
+  const addButtonText = uiSchema["ui:addButtonText"] || "Add";
   const fullWidth = !!uiSchema["ui:fullWidth"];
   const showLabel = uiSchema["ui:showLabel"] !== false;
+  const topAlignDelete = uiSchema["ui:topAlignDelete"] ?? false;
+  const isDisabled = disabled || readonly;
   return (
     <>
       {showLabel && (
```

Comment:
(This is for line `103` not `102` but I can't highlight that line 😭)

I think `TitleField` here is missing the new props `isSectionTitle`? I see it `undefined` when I try to print it. Alternatively could make `isSectionTitle` false by default in `CustomFields.tsx `😊

<img width="1429" alt="Screen Shot 2022-01-20 at 10 21 45 AM" src="https://user-images.githubusercontent.com/47064971/150368951-55a8d2b7-e663-49ae-ad24-62f34696af55.png">